stage1/init/common/seccomp_wildcards.go

```diff
@@ -0,0 +1,398 @@
+		"listxattr",
+		"llistxattr",
+		"_llseek",
+		"lremovexattr",
+		"lseek",
+		"lsetxattr",
+		"lstat",
+		"lstat64",
+		"madvise",
+		"memfd_create",
+		"mincore",
+		"mkdir",
+		"mkdirat",
+		"mknod",
+		"mknodat",
+		"mmap",
+		"mmap2",
+		"mprotect",
+		"mq_getsetattr",
+		"mq_notify",
+		"mq_open",
+		"mq_timedreceive",
+		"mq_timedsend",
+		"mq_unlink",
+		"mremap",
+		"msgctl",
+		"msgget",
+		"msgrcv",
+		"msgsnd",
+		"msync",
+		"munlock",
+		"munlockall",
+		"munmap",
+		"nanosleep",
+		"newfstatat",
+		"_newselect",
+		"open",
+		"openat",
+		"pause",
+		"personality", // this is args-filtered by docker
+		"pipe",
+		"pipe2",
+		"poll",
+		"ppoll",
+		"prctl",
+		"pread64",
+		"preadv",
+		"prlimit64",
+		"pselect6",
+		"pwrite64",
+		"pwritev",
+		"read",
+		"readahead",
+		"readlink",
+		"readlinkat",
+		"readv",
+		"recv",
+		"recvfrom",
+		"recvmmsg",
+		"recvmsg",
+		"remap_file_pages",
+		"removexattr",
+		"rename",
+		"renameat",
+		"renameat2",
+		"restart_syscall",
+		"rmdir",
+		"rt_sigaction",
+		"rt_sigpending",
+		"rt_sigprocmask",
+		"rt_sigqueueinfo",
+		"rt_sigreturn",
+		"rt_sigsuspend",
+		"rt_sigtimedwait",
+		"rt_tgsigqueueinfo",
+		"sched_getaffinity",
+		"sched_getattr",
+		"sched_getparam",
+		"sched_get_priority_max",
+		"sched_get_priority_min",
+		"sched_getscheduler",
+		"sched_rr_get_interval",
+		"sched_setaffinity",
+		"sched_setattr",
+		"sched_setparam",
+		"sched_setscheduler",
+		"sched_yield",
+		"seccomp",
+		"select",
+		"semctl",
+		"semget",
+		"semop",
+		"semtimedop",
+		"send",
+		"sendfile",
+		"sendfile64",
+		"sendmmsg",
+		"sendmsg",
+		"sendto",
+		"setfsgid",
+		"setfsgid32",
+		"setfsuid",
+		"setfsuid32",
+		"setgid",
+		"setgid32",
+		"setgroups",
+		"setgroups32",
+		"setitimer",
+		"setpgid",
+		"setpriority",
+		"setregid",
+		"setregid32",
+		"setresgid",
+		"setresgid32",
+		"setresuid",
+		"setresuid32",
+		"setreuid",
+		"setreuid32",
+		"setrlimit",
+		"set_robust_list",
+		"setsid",
+		"setsockopt",
+		"set_thread_area",
+		"set_tid_address",
+		"setuid",
+		"setuid32",
+		"setxattr",
+		"shmat",
+		"shmctl",
+		"shmdt",
+		"shmget",
+		"shutdown",
+		"sigaltstack",
+		"signalfd",
+		"signalfd4",
+		"sigreturn",
+		"socket",
+		"socketcall",
+		"socketpair",
+		"splice",
+		"stat",
+		"stat64",
+		"statfs",
+		"statfs64",
+		"symlink",
+		"symlinkat",
+		"sync",
+		"sync_file_range",
+		"syncfs",
+		"sysinfo",
+		"syslog",
+		"tee",
+		"tgkill",
+		"time",
+		"timer_create",
+		"timer_delete",
+		"timerfd_create",
+		"timerfd_gettime",
+		"timerfd_settime",
+		"timer_getoverrun",
+		"timer_gettime",
+		"timer_settime",
+		"times",
+		"tkill",
+		"truncate",
+		"truncate64",
+		"ugetrlimit",
+		"umask",
+		"uname",
+		"unlink",
+		"unlinkat",
+		"utime",
+		"utimensat",
+		"utimes",
+		"vfork",
+		"vmsplice",
+		"wait4",
+		"waitid",
+		"waitpid",
+		"write",
+		"writev",
+		"arch_prctl",
+		"modify_ldt",
+		"chroot",
+		"clone", // this is args-filtered by docker
```

Comment:
seccomp filters (in kernel) allows to match on syscall arguments too, but seccomp filters (in systemd) do not. The note is there as we are dropping the args-filtering part originally present in docker.